recipes/dsfmt/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+export CFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+export CPATH=${PREFIX}/include
+
+make std
```

Comment:
What does this do? Is there a general `make` call for building?

Reply:
Simply calling `make` has the same result as calling `make std`. It builds an executable called `test-std-M19937`

Reply:
I see. So guess we need to build and install the library ourselves.